src/lib/sidebar/EntireScheme.svelte

```diff
@@ -9,16 +9,22 @@
     openFromSidebar,
   } from "../../stores";
   import type { Scheme, Schema } from "../../types";
+  import ConfirmationModal from "../common/ConfirmationModal.svelte";
+  import { currentMode } from "../../stores";
```

Comment:
You can include this in the list of imports from the same file a few lines up.

I really wish prettier (or similar) would auto-format / order / group imports...